fuzzers/051-pip-imuxlout-bypalts/generate.tcl

```diff
@@ -62,8 +102,14 @@ for {set idx 0} {$idx < [llength $todo_lines]} {incr idx} {
 
     set mynet [create_net mynet_$idx]
     set mylut [create_cell -reference LUT1 mylut_$idx]
+    set dst_type [regsub {.*(.$)} $clb_dst_pin {\1}]
     set lutin [regsub {.*(.)} $clb_dst_pin {A\1}]
-    set_property -dict "LOC $slice BEL $lut LOCK_PINS I0:$lutin" $mylut
+
+    if { $dst_type >= 0 && $dst_type <= 6 } {
+        set_property -dict "LOC $slice BEL $lut LOCK_PINS I0:$lutin" $mylut
+    } else {
+        set_property -dict "LOC $slice BEL $lut" $mylut
```

Comment:
Can you add a comment on what this is doing?

Reply:
done